ArmoniK.Extension.CSharp.DllCommon/IDllWorker.cs

```diff
@@ -0,0 +1,29 @@
+// This file is part of the ArmoniK project
+// 
+// Copyright (C) ANEO, 2021-2024. All rights reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License")
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using ArmoniK.Api.gRPC.V1;
+using ArmoniK.Api.Worker.Worker;
+
+using Microsoft.Extensions.Logging;
+
+namespace ArmoniK.Extension.CSharp.DllCommon;
+
+public interface IDllWorker
```

Comment:
I think we can get a better name for this class. A class more targeted to function and waht we want to do